src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -558,8 +559,8 @@ class GoogleExtensions(
     }
 
     def checkPubsub: Future[SubsystemStatus] = {
-      logger.debug("Checking Google PubSub...")
-      googlePubSubDAO.getTopic(googleServicesConfig.groupSyncTopic).map {
+      logger.debug("Checking Google Group Sync PubSub...")
+      googleGroupSyncPubSubDAO.getTopic(googleServicesConfig.groupSyncTopic).map {
```

Comment:
Wasn't sure what to do for this one? Should we checkPubsub for the other projects as well?

Reply:
Seems like we should yeah since I think this gets reported in status